source/net/ServerSocket.ooc

```diff
@@ -118,22 +114,22 @@ ServerSocket: class extends Socket {
 
 		This method will block.
 	*/
-	accept: func ~withClosure (f: func (TCPServerReaderWriterPair) -> Bool) {
-		if (!listening)
-			listen()
+	accept: func ~withClosure (f: Func (TCPServerReaderWriterPair) -> Bool) {
+		if (!this listening)
+			this listen()
 
-		loop(||
+		while (true) {
```

Comment:
I don't know if `loop` was a keyword that disappeared or what was going on here?